src/Overlay.tsx

```diff
@@ -33,6 +34,20 @@ const widthMap = {
   auto: 'auto'
 }
 
+function getSlideAnimationFactors(anchorSide?: AnchorSide): {x: number; y: number} {
```

Comment:
Nit: Is “animation factors” a widely-used/understood phrase? If not, is there a more descriptive name for these values?

Reply:
Not at all common, naming is just hard 😄.  Happy to discuss alternatives

Reply:
Updated in https://github.com/primer/components/pull/1322/commits/db69e072593a12c20aece971e81a98840b0448f3